packages/create-turbo/__tests__/cli.test.ts

```diff
@@ -38,135 +63,193 @@ describe("create-turbo cli", () => {
+        `? Where would you like to create your turborepo? (./${DEFAULT_APP_NAME})`
+      );
+
+      expect(getPromptChoices(messages[2])).toEqual(["npm", "pnpm", "yarn"]);
+
+      expect(messages[3]).toMatch(
+        /^>>> Bootstrapped a new turborepo with the following:/
+      );
+
+      expect(getGeneratedPackageJSON().packageManager).toMatch(/^npm/);
+
+      expect(fs.existsSync(path.join(testDir, "node_modules"))).toBe(false);
+    }, 10000);
+
+    PACKAGE_MANAGERS.forEach((packageManager) => {
+      it(`--use-${packageManager}: guides the user through the process`, async () => {
+        const cli = spawn(
+          "node",
+          [createTurbo, "--no-install", `--use-${packageManager}`],
+          {}
+        );
+        const messages = await runInteractiveCLI(cli);
+
+        expect(messages[0]).toEqual(
+          ">>> Welcome to Turborepo! Let's get you set up with a new codebase."
+        );
+
+        expect(messages[1]).toEqual(
+          `? Where would you like to create your turborepo? (./${DEFAULT_APP_NAME})`
+        );
+
+        expect(messages[2]).toMatch(
+          /^>>> Bootstrapped a new turborepo with the following:/
+        );
+
+        expect(getGeneratedPackageJSON().packageManager).toMatch(
+          new RegExp(`^${packageManager}`)
+        );
+
+        expect(fs.existsSync(path.join(testDir, "node_modules"))).toBe(false);
+      }, 10000);
     });
+  });
 
-    cli.on("exit", () => {
-      try {
-        done();
-      } catch (error) {
-        done(error);
-      }
-      return;
+  describe("with installation", () => {
+    it("default", async () => {
+      const cli = spawn("node", [createTurbo], {});
+
+      const messages = await runInteractiveCLI(cli);
+
+      expect(messages[0]).toEqual(
+        ">>> Welcome to Turborepo! Let's get you set up with a new codebase."
+      );
+
+      expect(messages[1]).toEqual(
+        `? Where would you like to create your turborepo? (./${DEFAULT_APP_NAME})`
+      );
+
+      expect(getPromptChoices(messages[2])).toEqual(["npm", "pnpm", "yarn"]);
+
+      expect(messages[3]).toMatch(
+        /^>>> Creating a new turborepo with the following:/
+      );
+
+      expect(getGeneratedPackageJSON().packageManager).toMatch(/^npm/);
+
+      expect(fs.existsSync(path.join(testDir, "node_modules"))).toBe(true);
+    }, 50000);
+
+    PACKAGE_MANAGERS.forEach((packageManager) => {
+      it(`--use-${packageManager}`, async () => {
+        const cli = spawn("node", [createTurbo, `--use-${packageManager}`], {});
+        const messages = await runInteractiveCLI(cli);
+
+        expect(messages[0]).toEqual(
+          ">>> Welcome to Turborepo! Let's get you set up with a new codebase."
+        );
+
+        expect(messages[1]).toEqual(
+          `? Where would you like to create your turborepo? (./${DEFAULT_APP_NAME})`
+        );
+
+        expect(messages[2]).toMatch(
+          /^>>> Creating a new turborepo with the following:/
```

Comment:
Can we add a check for:

`>>> Success! Created a new Turborepo at "${DEFAULT_APP_NAME}".`

as well? This would catch that `pnpm` error we had a few weeks ago (the install actually worked, pnpm was just non-zero exiting because it was upset about peers. (which caused us to not show this message and show `install has failed`

Reply:
I can add that, but I wasn't able to reproduce that failure, for what it's worth :/

Reply:
after this PR when tests are running in CI, we can have a bit more confidence about adding new tests if we have regressions again.